twill-api/src/main/java/org/apache/twill/api/TwillRunResources.java

```diff
@@ -58,7 +60,18 @@
 
   /**
    * @return the enabled log level for the container where the runnable is running in.
+   * @deprecated Use {@link #getRootLogLevel()} instead.
    */
+  @Deprecated
   LogEntry.Level getLogLevel();
 
+  /**
+   * @return the enabled log level for the container where the runnable is running in.
+   */
+  LogEntry.Level getRootLogLevel();
```

Comment:
Same reason as above, don't introduce this method just specialized for root logger.